dev/validate-examples/main.go

```diff
@@ -0,0 +1,117 @@
+package main
```

Comment:
it's probably a good idea to add separated go.mod here, to not affect entire package